rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -1175,6 +1176,10 @@ public void visitConstantExpression(ConstantExpression expression) {
                     jType = JavaType.Primitive.String;
                     // String literals value returned by getValue()/getText() has already processed sequences like "\\" -> "\"
                     int length = sourceLengthOfNext(expression);
+                    // this is an attribute selector
+                    if (source.startsWith("@"+value, cursor)) {
```

Comment:
This feels like it should be represented in the Groovy AST in some form to indicate that the current identifier is a specialized one using the attribute operator.

The way that these operators are typically represented is by the usage of a `Marker` type on the `J.Identifier`, but let me double check the Groovy AST node here first to confirm this suspicion.

Reply:
I think I'm fine with this being calculable state on the identifier without requiring a marker.